lib/line-index/src/lib.rs

```diff
@@ -136,6 +136,22 @@ impl LineIndex {
         self.start_offset(line_col.line as usize).map(|start| start + TextSize::from(line_col.col))
     }
 
+    /// Transforms the `LineCol` into a `TextSize`. If the column exceeds the line length,
+    /// the line, it is clamped to that.
```

Comment:
Might be good to describe the return types here then, its not immediately obvious what they mean